assignment_day_11/src/components/Dialog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {View, Text, StyleSheet, Modal, ActivityIndicator} from 'react-native';
+import colors from '../constants/colors';
+
+const Dialog = ({visibility}) => {
+  return (
+    <Modal visible={visibility}>
+      <View style={styles.outerDialog}>
+        <View style={styles.dialog}>
+          <Text style={styles.dialogTitle}>Loading...</Text>
+          <ActivityIndicator size="large" color={colors.loaderColor} />
+        </View>
+      </View>
+    </Modal>
+  );
+};
+
+const styles = StyleSheet.create({
+  outerDialog: {
+    flex: 1,
+    backgroundColor: '#00000020',
```

Comment:
you already created constants for colors? don't you think you should refer to the constant file

Reply:
Apologies did it in a hurry at that time.
Forgot to add it to the constants file